dcc-storage-client/pom.xml

```diff
@@ -130,6 +130,12 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+	<dependency>
+      <groupId>commons-io</groupId>
```

Comment:
Adding new dependencies for 1 or 2 utility methods should be avoided. We already have many and this just increases the likelihood of conflicts.